source/dashboards/badges.markdown

```diff
@@ -10,6 +10,66 @@ Badges are widgets that sit at the top of a panel, above all the cards.
   Badges at the top of a panel.
 </p>
 
+## Entity Badge
+
+The Entity badge allows you to display the state of an entity on a badge. This badge supports [actions](/dashboards/actions/).
+
+```yaml
+type: entity
+entity: light.living_room
+```
+
+{% configuration entity %}
+type:
+  required: true
+  description: "`entity`"
+  type: string
+entity:
+  required: true
+  description: Entity ID.
+  type: string
+name:
+  required: false
+  description: Overwrites the entity name.
+  type: string
+icon:
+  required: false
+  description: Overwrites the entity icon.
+  type: string
+color:
+  required: false
+  description: Set the color when the entity is active. By default, the color is based on `state`, `domain`, and `device_class` of your entity. It accepts [color token](/dashboards/tile/#available-colors) or hex color code.
+  type: string
+  default: state
+show_entity_picture:
+  required: false
+  description: If your entity has a picture, it will replace the icon.
+  type: boolean
+  default: false
+display_type:
+  required: false
+  description: Type of display for the badge. It can be either `minimal` (icon only), `standard` (icon and state) or `complete` (icon, state and name).
```

Comment:
```suggestion
  description: Type of display for the badge. It can be either `minimal` (icon only), `standard` (icon and state), or `complete` (icon, state, and name).
```